main.c

```diff
@@ -325,6 +325,23 @@ int force_dfu_gpio() {
 #define RCC_CR       (*(volatile uint32_t*)0x40021000U)
 #define RCC_CFGR     (*(volatile uint32_t*)0x40021004U)
 
+#define RCC_CSR      (*(volatile uint32_t*)0x40021024U)
+#define RCC_CSR_LPWRRSTF			(1 << 31)
+#define RCC_CSR_WWDGRSTF			(1 << 30)
+#define RCC_CSR_IWDGRSTF			(1 << 29)
+#define RCC_CSR_SFTRSTF			(1 << 28)
+#define RCC_CSR_PORRSTF			(1 << 27)
+#define RCC_CSR_PINRSTF			(1 << 26)
+#define RCC_CSR_RMVF				(1 << 24)
```

Comment:
Can you use spaces here perhaps? Make is more readable.

Reply:
Ok.